docs/guide/installation/advanced.rst

```diff
@@ -92,17 +92,17 @@ SunPy has the following strict requirements:
 
 - `Python <https://www.python.org/>`__ 3.6.x or later.
 
-- `NumPy <https://www.numpy.org/>`__  1.14.5 or later.
+- `NumPy <https://www.numpy.org/>`__  1.15.0 or later.
 
 - `SciPy <https://www.scipy.org/>`__ 1.0.0 or later.
 
 - `Astropy <https://www.astropy.org/>`__ 3.2.0 or later.
 
-- `pandas <https://pandas.pydata.org/>`__ 0.10 or later.
+- `pandas <https://pandas.pydata.org/>`__ 0.23.0 or later.
```

Comment:
I wonder if we should pin to 1.0 instead.

Reply:
Is there a good reason for this? I'm still personally trying to work breaking changes in pandas 1.0, and it was released fairly recently.

Reply:
I assumed it was a LTS release for pandas, but i never checked.